### PR TITLE
chore(doc): doc for managing the contabo server and rancher

### LINKED ISSUE #649: How to manage Contabo server and Rancher
Ensure that both the Rancher and Contabo Kubernetes servers are up to date and functioning properly.

----------

-  What do you have in mind for node configuration for the test a prod clusters?
-  How many master and worker nodes in each cluster?
-

### PR DESCRIPTION
**Pull request by maxyzli**
resolves #649

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ Now, pre-commit will run automatically on `git commit`. If you want to manually
 
 For a complete guide on how to deploy and run your services on Contabo VPS, please follow the instructions in our dedicated tutorial:
 
-[Run Murmurations Services in Contabo](docs/contabo/run-in-contabo.md)
+[Run Murmurations Services in Contabo](docs/contabo/run-murmurations-services-on-contabo.md)
```
Comment (maxyzli) [inline README.md:92]: /rebuild

Comment (maxyzli) [inline README.md:92]: /rebuild

Comment (maxyzli) [inline README.md:92]: /rebuild

Comment (maxyzli) [inline README.md:92]: /rebuild

Comment (maxyzli) [inline README.md:92]: /rebuild

File: README.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ Now, pre-commit will run automatically on `git commit`. If you want to manually
...
 
 ## Run in DigitalOcean
```
Comment (maxyzli) [inline README.md:94]: /rebuild

File: README.md (hunk 1 above)
Comment (maxyzli) [inline README.md:92]: /rebuild

**Review by geoffturk — APPROVED** [review]
LGTM 🚀

**Pull request merged**